chart/files/statsd-mappings.yml

```diff
@@ -52,6 +52,13 @@ mappings:
       dag_id: "$1"
       task_id: "$2"
 
+  - match: airflow.dag.*.*.*.duration
```

Comment:
You can use dots in normal task_ids too. This doesn't mean the task is in a task group.

Reply:
So the 2 dots will not be able to glob those tasks either then?

Reply:
Another curve ball, guess what? Dags can have dots too :) So imagine we have `airflow.dag.first.second.third`. This could be:

- dag_id=first, task_group=second, task_id=third
- dag_id=first.second, task_id=third (no task_group at all)
- dag_id=first, task_id=second.third (no task_group at all)

You can also disable the task group id prefixing with `prefix_group_id=False`, so those last 2 could actually be in a task group, but not one named `second`.

Basically, there isn't a reliable way to extract task group id out.